pkg/cli/cmd/bicep/publish/publish.go

```diff
@@ -22,17 +22,15 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"net/http"
 	"strings"
 
-	credentials "github.com/oras-project/oras-credentials-go"
```

Comment:
This was deprecated and I added `oras.land/oras-go/v2/registry/remote/credentials` instead of that. The changes in this file just represent that and the change of the order of the imports.